person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [ ] **Optional:** Note that conventionally, `attr`s are placed at the top of the class before the `initialize` method.

Reply:
I've done the changes thanks for the suggestion again! 💯